src/test/java/BaseTest.java

```diff
@@ -1,11 +1,229 @@
+//                firefoxOptions.addArguments("--remote-allow-origins=*");
+//                return driver = new FirefoxDriver(firefoxOptions);
+//
+//            case "MicrosoftEdge":
+//                WebDriverManager.edgedriver().setup();
+//                EdgeOptions edgeOptions= new EdgeOptions();
+//                edgeOptions.addArguments("--remote-allow-origins=*");
+//                return driver = new EdgeDriver(edgeOptions);
+//
+//            case "grid-edge": //gradle clean test -Dbrowser=grid-edge
+//                caps.setCapability("browserName", "MicrosoftEdge");
+//                return driver = new RemoteWebDriver(URI.create(gridUrl).toURL(), caps);
+//
+//            case "grid-firefox": //gradle clean test -Dbrowser=grid-firefox
+//                caps.setCapability("browserName", "firefox");
+//                return driver = new RemoteWebDriver(URI.create(gridUrl).toURL(), caps);
+//
+//            case "grid-chrome": //gradle clean test -Dbrowser=grid-chrome
+//                caps.setCapability("browserName", "chrome");
+//                return driver = new RemoteWebDriver(URI.create(gridUrl).toURL(), caps);
+//
+//            case "cloud":    //gradle clean test -Dbrowser=grid-cloud
+//                return lambDaTest();
+//
+//            default:
+//                WebDriverManager.chromedriver().setup();
+//                ChromeOptions chromeOptions = new ChromeOptions();
+//                chromeOptions.addArguments("--remote-allow-origins=*");
+//                return driver = new ChromeDriver(chromeOptions);
+//        }
+//    }
+//
+//    public WebDriver lambDaTest() throws MalformedURLException{
+//        String hubUrl = "https://hun.lambdatest.com/wd/hub";
+//
+//        DesiredCapabilities capabilities = new DesiredCapabilities();
+//
+////        ChromeOptions browserOptions = new ChromeOptions();
+////        browserOptions.setPlatformName("Windows 11");
+////        browserOptions.setBrowserVersion("121.0");
+//
+//        capabilities.setCapability("browserName", "chrome");
+//        capabilities.setCapability("browserVersion", "121.0");
+//
+//        HashMap<String, Object> ltOptions = new HashMap<String, Object>();
+//        ltOptions.put("username", "magiclogiclu");
+//        ltOptions.put("accessKey", "JLZPlaaZpjpkOofHN9yfGXrCXUO0GkRWCaX854MPEyjHawNwNv");
+//        ltOptions.put("geoLocation", "CA");
+//        ltOptions.put("visual", true);
+//        ltOptions.put("video", true);
+//        ltOptions.put("project", "Untitled");
+//        ltOptions.put("selenium_version", "4.5.0");
+//        ltOptions.put("driver_version", "121.0");
+//        ltOptions.put("w3c", true);
+//        ltOptions.put("plugin", "java-testNG");
+////        browserOptions.setCapability("LT:Options", ltOptions);
+//        capabilities.setCapability("LT:Options", ltOptions);
+//
+//        return new RemoteWebDriver(new URL(hubUrl), capabilities);
+//    }
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.